### PR TITLE
Update README to include Cloudfont pretty url config

### PR DESCRIPTION
**Pull request by jarredkenny**
Modified documentation in README.md to list both Cloudfront and Static Website Hosting options for supporting Pretty URLs on S3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,7 +226,25 @@ The following is an example policy that meets these requirements:
 
 
 ## Using History-Location
-You can deploy your Ember application to S3 and still use the history-api for pretty URLs. This needs some configuration tweaking in your bucket's static-website-hosting options in the AWS console though. You can use S3's `Redirection Rules`-feature to redirect user's to the correct route based on the URL they are requesting from your app:
+You can deploy your Ember application to S3 and still use the history-api for pretty URLs. This needs some configuration and the exact process depends on wether or not you are using Cloudfront to serve cached content from your S3 bucket or if you are serving from an S3 bucket directly using S3's Static Website Hosting option. Both options work however the Cloudfront method allows the process to occur without flashing a non-pretty URL in the browser before the application loads.
```
Comment (achambers) [inline README.md:229]: s/wether/whether/

Comment (achambers) [inline README.md:229]: s/Both options work however the/Both options work, however, the/

**Comment by achambers** [comment]
Thanks @jarredkenny . Left 2 minor grammatical comments there.

Do you think there is any place for some documentation around this on the actual ember-cli-deploy docs site? Where did you go to first to try and understand this?

File: README.md [hunk 2]
```diff
@@ -242,6 +260,8 @@ You can deploy your Ember application to S3 and still use the history-api for pr
 </RoutingRules>
 ```
 
+
+
```
Comment (ghedamat) [inline README.md:264]: nitpicking here but could you remove these whitelines if they're not needed?

**Pull request closed without merging**